mgmt/src/models/external/configdb/gwconfig.rs

```diff
@@ -18,7 +18,7 @@ use crate::frr::frrmi::FrrMi;
 use crate::processor::confbuild::build_internal_config;
 
 /// Alias for a config generation number
-pub type GenId = u64;
+pub type GenId = i64;
```

Comment:
Why can't this be u64? Or, what's the advantage of using i64?

Reply:
This is a reflection of what has been done to reflect changes in gateway-proto.
But in particular, this change is done to be consistent with K8S API.